<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickMergeSort</class>
 <widget class="QDialog" name="QuickMergeSort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>941</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1022</width>
    <height>941</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Исходный массив</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="input">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Выберете режим сортировки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="buttonQMergeSort">
     <property name="text">
      <string>Отсортировать</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Результат:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Следующий Шаг</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="resultText">
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSave">
     <property name="text">
      <string>Введите путь к текстовому файлу, в котором хотите сохранить результат</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineSave"/>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>Сохранить результат</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
